vcluster/configure/vcluster-yaml/policies/network-policy.mdx

```diff
@@ -44,24 +44,50 @@ policies:
     enabled: true
 ```
 
-#### Custom egress rules {#custom-egress}
-
-Control outbound traffic with specific CIDR blocks:
-
+#### Custom ingress and egress rules {#custom-rules}
+Control inbound and outbound traffic with specific ports and IP addresses for vCluster control plane and workloads:
```

Comment:
What happens when this is disabled (the default)? It would help users to know that without network policies, vCluster pods can communicate freely with all pods on the host cluster.